go/cmd/vtgateclienttest/services/success.go

```diff
@@ -32,11 +32,11 @@ func (c *successClient) Begin(ctx context.Context) (*vtgatepb.Session, error) {
 	}, nil
 }
 
-func (c *successClient) Commit(ctx context.Context, session *vtgatepb.Session) error {
+func (c *successClient) Commit(ctx context.Context, twopc bool, session *vtgatepb.Session) error {
```

Comment:
Shouldn't this file have a ResolveTransaction function too?

And it should be exercised by the various clients, right? Like Java, PHP, python, ... Or is this not wired yet, but is coming soon?